src/standalone/Vegetation/PlantHydraulics.jl

```diff
@@ -370,6 +371,15 @@ function water_flux(z1::FT, z2::FT, ψ1::FT, ψ2::FT, K1::FT, K2::FT) where {FT}
     return flux # (m/s)
 end
 
+"""
+    root_distribution(z::FT, rooting_depth::FT)
+
+Compute value of rooting probabilty density function at `z`
```

Comment:
Could you sketch out here the connection between rooting depth and beta?
E.g. give the CDF F(z) = 1-beta^(100z) and cite Equation 2.23 of Bonan, "Climate Change and Terrestrial Ecosystem Modeling", 2019 Cambridge University Press. And say this is equivalent to this probability density function with coordinate `z` replacing depth.